rental/src/main/java/pl/edu/agh/rental/rental/RentalController.java

```diff
@@ -45,31 +45,17 @@ public RentalData getRental(@CurrentUser final User user) throws NoRentalError {
         return reservationService.getRental(user);
     }
 
-    //current time, current rental
-    @GetMapping("/time/current")
-    @ResponseStatus(HttpStatus.OK)
-    public long getCurrentRentalTime(@CurrentUser final User user) throws NoRentalError {
-        return reservationService.getCurrentRentalTime(user);
-    }
-
-    //total time after end
-    @GetMapping("/time/{id}")
-    @ResponseStatus(HttpStatus.OK)
-    public long getRentalTime(@PathVariable("id") final Integer reservationId) throws UserUnauthorizedError{
-        return reservationService.getRentalTime(reservationId);
-    }
-
     //current cost - current rental
     @GetMapping("/cost/current")
     @ResponseStatus(HttpStatus.OK)
-    public long getCurrentRentalCost(@CurrentUser final User user) throws NoRentalError {
+    public double getCurrentRentalCost(@CurrentUser final User user) throws NoRentalError {
```

Comment:
It should remain long as we are displaying the value*100(w groszach). For example when the cost is 10.78PLN, the returned value should be 1078.

Reply:
This should be long

Reply:
```suggestion
    public long getCurrentRentalCost(@CurrentUser final User user) throws NoRentalError {
```